### DISCUSSION TITLE
TabView + LazyView

### DISCUSSION THREAD

**Discussion by angelru** [xamarin/XamarinCommunityToolkit#1074] [Q&A] [answered]
I'm looking at the develop sample:
The LazyView ViewModel

public static LazyTestViewModel Current { get; } = new LazyTestViewModel();

It's already initialized, but shouldn't it be created when the view is also created?

**Comment by pictos** [comment, marked as answer]
In this specific case I don't want that, I want to have a ViewModel pre-initialized to share between pages.

**Reply by angelru** [reply]
but if it is not pre-initialized, is the data not binded?

**Reply by pictos** [reply]
No, VM and the View are two different things, I want to have the VM created before any View. But that VM will be the BindingContext of the LazyView when its constructor is called. You can clone this repo and run the sample, add a breakpoint inside the LazyView ctor and you will see that even if the VM is created the View isn't
